lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -118,6 +120,9 @@ def sync(mediator)
       @instance_data = mediator.get_instance(kubectl_resource_type, name)
     end
 
+    def post_sync
```

Comment:
Note: try to think of a better name for this

Reply:
maybe `after_sync`? that's maybe a bit more in line with how other popular Ruby frameworks call lifecycle hooks like this (`after_action` in Rails, `after_enqueue` in Resque, ...).